scheduler/test/cook/test/mesos/task.clj

```diff
@@ -346,7 +347,8 @@
                         :extract false
                         :value "file:///path/to/cook-executor"}}
         mesos-run-as-user nil]
-    (with-redefs [cook.config/executor-config (constantly executor)]
+    (with-redefs [cook.config/executor-config (constantly executor)
+                  cc/container-defaults (constantly {})]
```

Comment:
Nitpick: Should we define `compute-cluster (Object.)` and validate it is the argument received in the `cc/container-defaults` mocked functions?

Reply:
Seems like overkill to me